### PR TITLE
Fe profile photos upload

### PR DESCRIPTION
**Pull request by xia0m**
### What this PR does (required):
- Created the UI for the uploading/editing profile photo under profile settings
- When a photo is selected from file browser, a preview of that photo is displayed

### Screenshots / Videos (required):
CleanShot 2021-05-24 at 01 31 46@2x
[GIF](https://user-images.githubusercontent.com/6465981/119320364-5e364880-bc30-11eb-8bf6-2043fcf48ab5.gif)

### Any information needed to test this feature (required):
- Sign up and login
- Click profile photo on the top right corner
- Click profile
- Click Profile Photo

### CODE REVIEW

File: client/src/components/Account/UploadPhoto/UploadPhoto.tsx [hunk 1]
```diff
@@ -1,12 +1,88 @@
-import { Box, Typography } from '@material-ui/core';
+import { Box, Typography, Avatar, Grid, Button, FormControl } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import SaveIcon from '@material-ui/icons/Save';
+
+import React, { ChangeEventHandler, useRef, useState } from 'react';
```
Comment (dproc96) [inline client/src/components/Account/UploadPhoto/UploadPhoto.tsx]:
```suggestion
import React, { useRef, useState } from 'react';
```
You can remove this unused import

File: client/src/components/Account/UploadPhoto/UploadPhoto.tsx [hunk 2]
```diff
@@ -1,12 +1,88 @@
...
 
 import useStyles from './useStyles';
+import profile_url from '../../../Images/b1f0e680702e811aa8ba333cb19c0e0ea95e8e31.png';
+import { isNull } from 'util';
```
Comment (dproc96) [inline client/src/components/Account/UploadPhoto/UploadPhoto.tsx]:
```suggestion
```
This one too

File: client/src/components/Account/UploadPhoto/UploadPhoto.tsx [hunk 3]
```diff
@@ -1,12 +1,88 @@
...
+                    Upload a file from your device
+                  </Box>
+                </Button>
+              </Grid>
+              <Grid item>
+                <Grid container direction="row" justify="space-around">
+                  <Grid>
+                    <Button color="secondary" startIcon={<SaveIcon style={{ color: 'black' }} />}>
+                      Save Photo
+                    </Button>
```
Comment (dproc96) [inline client/src/components/Account/UploadPhoto/UploadPhoto.tsx]: I think this button is adding an extra step that is weird from a UX perspective. If it's to match the mocks then you can ignore my comment for now and figure out later in implementation. If you do keep it, I think it should change color or something when you've uploaded the image so the user knows to click it to complete the update

**Review by dproc96 — APPROVED** [review]
This looks good to go, I had two very small change requests and one thought for the future. You can go ahead and merge when the change requests are done

File: client/src/components/Account/UploadPhoto/UploadPhoto.tsx (hunk 3 above)
Comment (xia0m) [reply client/src/components/Account/UploadPhoto/UploadPhoto.tsx]: No problem, I deleted the save button

**Pull request merged**